src/pages/StudyUpdateForm.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import { useCallback, useState, useEffect } from 'react';
+import { Notifications } from '../libs/utils';
+import lockIcon from '../images/lock-icon.png';
+import { Styles } from '../libs/theme';
+import { cloneDeep, isNil, isEmpty } from 'lodash/fp';
+
+import DataSubmissionStudyInformation from '../components/data_submission/ds_study_information';
+import NihAnvilUse from '../components/data_submission/NihAnvilUse';
+import NIHAdministrativeInformation from '../components/data_submission/NIHAdministrativeInformation';
+import NIHDataManagement from '../components/data_submission/NIHDataManagement';
+import DataAccessGovernance from '../components/data_submission/DataAccessGovernance';
+import { DataSet, User, Institution } from '../libs/ajax';
+import {Storage} from '../libs/storage';
+import { set } from 'lodash';
+
+
+export const StudyUpdateForm = (props) => {
+  const { history } = props;
+  const { studyId } = props.match.params;
+
+  const [formData, setFormData] = useState({properties:{}});
+  const [study, setStudy] = useState({});
+  const [user, setUser] = useState({});
+  const [failedInit, setFailedInit] = useState(false);
+  const [institutions, setInstitutions] = useState([]);
+  const [allConsentGroupsSaved, setAllConsentGroupsSaved] = useState(false);
+  const [formFiles, setFormFiles] = useState({});
+  const studyEditMode = true;
+
+  useEffect(() => {
+    const getAllInstitutions = async() => {
+      const institutions = await Institution.list();
+      setInstitutions(institutions);
+    };
+
+    const init = async () => {
+      try {
+        getAllInstitutions();
+      } catch (error) {
+        setFailedInit(true);
+        Notifications.showError({
+          text: 'Error: Unable to initialize data from server',
+        });
+      }
+      try {
+        const me = Storage.getCurrentUser();
+        setUser(me);
+        setStudy(await DataSet.getStudyById(studyId));
+        setFailedInit(false);
+      } catch (error) {
+        Notifications.showError({ text: 'Failed to load study' });
+      }
```

Comment:
Since the default for `failedInit` is `false`, if the previous try/catch passes it will continue to be `false`, so the statement has no effect. I believe you want to check whether the init fails in the error handler instead.